src/main/java/org/spongepowered/api/item/ItemType.java

```diff
@@ -80,5 +82,15 @@
      * @return The item property, if available
      */
     <T extends Property<?, ?>> Optional<T> getDefaultProperty(Class<T> propertyClass);
+    
+    @Override
+    default ItemType getType() {
```

Comment:
@Zidane I agree, but I don't know of any better name (other than `getItemType()`, which would also cause confusion). IMHO `getGameDictionaryType()` seems weird because it's returning a type of item, not a type of game dictionary.